src/test/java/com/origamisoftware/teach/advanced/services/DatabasePersonServiceTest.java

```diff
@@ -0,0 +1,85 @@
+// package com.origamisoftware.teach.advanced.services;
+
+// import com.origamisoftware.teach.advanced.model.DatabaseAccessObjects.Person;
+// import com.origamisoftware.teach.advanced.model.DatabaseAccessObjects.PersonTest;
+// import com.origamisoftware.teach.advanced.model.DatabaseAccessObjects.Quotes;
+// import com.origamisoftware.teach.advanced.util.DatabaseUtils;
+// import org.junit.After;
+// import org.junit.Before;
+// import org.junit.Test;
+// import java.sql.Timestamp;
+// import java.util.Calendar;
+// import java.util.List;
+// import static org.junit.Assert.*;
+
+// /**
+//  * Unit tests for DatabasePersonService
+//  */
+// public class DatabasePersonServiceTest {
+//     private DatabasePersonService databasePersonService;
+
+//     private void initDb() throws Exception {
+//         DatabaseUtils.initializeDatabase(DatabaseUtils.initializationFile);
+//     }
+
+//     @Before
+//     public void setUp() throws Exception {
+//         initDb();
+//         databasePersonService = ServiceFactory.getPersonServiceInstance();
+//     }
+
+//     @After
+//     public void tearDown() throws Exception {
+//         initDb();
+//     }
+
+//     @Test
+//     public void getPerson() throws PersonServiceException {
+//         List<Person> peopleList = databasePersonService.getPerson();
+//         assertFalse("Make sure we get some Person objects from service", peopleList.isEmpty());
+//     }

```

Comment:
was hoping to see the circle ci config set up the db so you can run these tests. 
do not include commented out code